wolfcrypt/src/integer.c

```diff
@@ -1197,7 +1197,9 @@ int mp_invmod_slow (mp_int * a, mp_int * b, mp_int * c)
     goto LBL_ERR;
   }
   if (mp_isone(&x)) {
-    mp_set(c, 1);
+    if ((res = mp_set(c, 1)) != MP_OKAY) {
```

Comment:
Always going to LBL_ERR.
res is going to be MP_OKAY when operation worked.

change to, if this makes sense:
res = mp_set(c, 1);
goto LBL_ERR;

Reply:
Good point. Pushed a fix.